wiki/cloud_admin/project_management.md

```diff
@@ -0,0 +1,105 @@
+# Project Management
+
+## Project Applications
+
+For the selected facility the facility managers can both manage the current project applications and view the application history.
+
+## Applications for review
+
+!!! Info "Project types"
+    This section only holds OpenStack projects associated with the facility, as the SimpleVM projects are handled in a pool-project running in OpenStack. This pool-project is handled as an individual project application.
+
+The review area of the application management holds 5 tabs.
+Each of the projects listed in these tabs have been processed by the VO before they get listed here. 
+
+* Submitted Applications
```

Comment:
The Tabs  Names are not Headers or bigger in the result (also for the other headings)

![grafik](https://github.com/user-attachments/assets/6832e745-c4c0-4777-a4b3-bb97a5625025)

Reply:
"This tab the initial project applications that got approved the VO governance and got allocated to the corresponding facility" -- something is missing